cwsdk/cube/Creature.h

```diff
@@ -337,12 +337,12 @@ namespace cube {
 		int field_11D4;
 		int field_11D8;
 		STDVector Inventory_Tabs_Vec;
-		int field_11E8;
+		int selected_item_count;
```

Comment:
I prefer UNIX-style names. Perhaps go for `se_it_num`.

Reply:
I know that's a joke, but I really do like `lower_snake_case` for C/C++ identifiers. :P

This repo is an absolute disaster anyway though, it has mixed naming conventions, mixed types (`int` & `char` instead of a fixed-sized `int32_t` and `int8_t` in some places), I haven't fully tested that [overriding the global `new` and `delete`](https://github.com/Andoryuuta/cwsdk/blob/master/cwsdk/memory_management.cpp) works the way I except, etc. So just having these fields documented is an awesome improvement. 😄